pyworkflow/pyworkflow/NodeFactory.py

```diff
@@ -0,0 +1,37 @@
+from .node import *
```

Comment:
Python filename style is `node_factory.py` --- if you change this, make sure to adjust the two imports (`workflow.py` and `__init__.py`)